### ISSUE TITLE
Share your screenshots (with english text) to replace the ones in readme.

### ISSUE THREAD

**Issue by azagaya** [azagaya/laigter#19]
Right now, the screenshots in the README have text in spanish. They should be changed to screenshots with english text. Also, we should check out if there are options for showing localized content.

**Comment by creepertron95** [comment]
@azagaya Here is a screenshot of the default image. The first image with the skull. 

I'm not sure of what images to showcase at the moment.

Should the video on the webpage be redone as well?

![laigter-screen-1](https://user-images.githubusercontent.com/53346722/89237362-9cd31780-d5b8-11ea-9664-0d261bfda0cc.png)

**Comment by azagaya** [comment]
Hi thanks! I should look for images that look cool to take more screenshots. The idea is to showcase a bit of major features. I'll look for the previously used textures, and share them here. Perhaps gifs would be better to show some features like tile-x and tile-y, brushes, and so. 
About the video, i was meant to replace it once i finish writing documentation, so we can decide what we put on it. What do you think?

**Comment by azagaya** [comment]
![stones](https://user-images.githubusercontent.com/46932830/89288547-35ef4600-d62c-11ea-8954-fbfbe29bbcba.png)
![tree1](https://user-images.githubusercontent.com/46932830/89288555-38ea3680-d62c-11ea-8ba8-1f859aa77d04.png)
![bricks2](https://user-images.githubusercontent.com/46932830/89288559-3a1b6380-d62c-11ea-868a-a91ec34606ce.jpg)
this are some of the images i used back then.. the tree and stones were made by me (programmer's art, dont judge!). The other one was taken from learnopengl, and its good to show parallax effect.

**Comment by creepertron95** [comment]
@azagaya I think I have an idea to find more images. You could have the community (Discord, Twitter, etc.) show off their work in the new version for screenshots.

**Comment by azagaya** [comment]
Yes, i asked them some time ago, but everybody told me they had nothing ready to show. Ill porbably ask again. Perhaps we could use sprites of my own wip game also. I'll ask the artist if he is ok with that.

**Comment by creepertron95** [comment]
@azagaya I took another screenshot to show off the Pixelated feature.

![laigter-screen-2](https://user-images.githubusercontent.com/53346722/89798139-6e41c900-daf1-11ea-93fa-556d8c299884.png)

I created the sprite with Aseprite.
![azagaya](https://user-images.githubusercontent.com/53346722/89798181-7bf74e80-daf1-11ea-927c-6f971f2bbfc2.png)

**Comment by azagaya** [comment]
Cool thanks a lot! Ill include it soon in the readme!

**Comment by creepertron95** [comment]
@azagaya I can be able to upload the pictures. I had to reset my internet. This should pretty much cover all of the current screenshots on gihtub.

![laigter-screen-3](https://user-images.githubusercontent.com/53346722/89803516-8cf78e00-daf8-11ea-8e5d-56dffd6fc408.png)
![laigter-screen-4](https://user-images.githubusercontent.com/53346722/89803543-95e85f80-daf8-11ea-8738-9b85d3d2cfb0.png)
![laigter-screen-5](https://user-images.githubusercontent.com/53346722/89803550-997be680-daf8-11ea-9925-9e8b4133a69f.png)

**Comment by creepertron95** [comment]
@azagaya Do you still have this image that is featured on your itch io page?
![Uxa_Sd](https://user-images.githubusercontent.com/53346722/89805120-c9c48480-dafa-11ea-9b7f-b461a036864d.png)

**Comment by azagaya** [comment]
I think i should have it.. ill look for it after work!

**Comment by azagaya** [comment]
Sorry, i was to busy yesterday, today ill look for that image and replace the screenshots.

**Comment by azagaya** [comment]
![cave-ground-with-stonepath-mid](https://user-images.githubusercontent.com/46932830/89948476-9407c480-dbfc-11ea-9e5d-1cc2b883c686.png)
![cave-ground-mid-mid](https://user-images.githubusercontent.com/46932830/89948479-9538f180-dbfc-11ea-9926-648f5bf48f5e.png)
Hi! this are those missing sprites.

**Comment by azagaya** [comment]
I updated the screenshots of the appdata xml file, and of the readme. This issue should remain open in case someone wants to share their own screenshots.
